### PR TITLE
HOCS-4716 BF2 Field auto population fix

### PR DESCRIPTION
**Pull request by robertdeniszczyc2**
Removes case summary, previous complaint reference, third party reference
from field auto population on BF2 case creation

### CODE REVIEW

File: src/test/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBfToBf2Test.java [hunk 1]
```diff
@@ -28,12 +28,12 @@
 
     static {
         FROM_CLOB = Map.of(
-                "CaseSummary", "TestValue");
+                "ComplaintReason1", "TestValue");
```
Comment (LiamMacP) [inline src/test/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBfToBf2Test.java:31]: `ComplaintReason1` related to this change?

Also sort of feels like you should have the caseSummary etc in this.

**Review by LiamMacP — COMMENTED** [review]
Just a couple of comments

File: src/test/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBfToBf2Test.java (hunk 1 above)
Comment (robertdeniszczyc2) [reply src/test/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBfToBf2Test.java:31]: Will also resolve what should be in FROM_CLOB

**Comment by robertdeniszczyc2** [comment]
> Just a couple of comments 🙂

Updated for comments, cheers!

**Review by LiamMacP — APPROVED** [review]
Changed handled and looks good - approved!

**Pull request merged**